native/navigation/route-names.js

```diff
@@ -97,6 +101,8 @@ export type MoreParamList = {|
   BuildInfo: void,
   DevTools: void,
   AppearancePreferences: void,
+  FriendList: { a: boolean },
```

Comment:
What's this for?

Reply:
Sorry, it was for testing purposes. I forgot to remove :(